Index: linux-3.18.23/drivers/net/Kconfig
===================================================================
--- linux-3.18.23.orig/drivers/net/Kconfig
+++ linux-3.18.23/drivers/net/Kconfig
@@ -360,4 +360,6 @@ config VMXNET3
 
 source "drivers/net/hyperv/Kconfig"
 
+source "drivers/net/rt_rdm/Kconfig"
+
 endif # NETDEVICES
Index: linux-3.18.23/drivers/net/Makefile
===================================================================
--- linux-3.18.23.orig/drivers/net/Makefile
+++ linux-3.18.23/drivers/net/Makefile
@@ -61,6 +61,8 @@ obj-$(CONFIG_VMXNET3) += vmxnet3/
 obj-$(CONFIG_XEN_NETDEV_FRONTEND) += xen-netfront.o
 obj-$(CONFIG_XEN_NETDEV_BACKEND) += xen-netback/
 
+obj-$(CONFIG_RALINK_RDM) += rt_rdm/
+
 obj-$(CONFIG_USB_NET_DRIVERS) += usb/
 
 obj-$(CONFIG_HYPERV_NET) += hyperv/
